src/audio/Kconfig

```diff
@@ -500,6 +500,13 @@ config PCM_CONVERTER_FORMAT_S16_C32_AND_S24_C32
 	  Support conversion between 16 bit valid sample size in 32 bit container
 	  and 24 bit valid sample size in 32 bit container
 
+config PCM_CONVERTER_FORMAT_S16_C32_AND_S16_C32
+	bool "Support S16C32 <-> S16C32"
+	default n
+	help
+	  Support conversion between 16 bit valid sample size in 32 bit container
+	  and 16 bit valid sample size in 32 bit container
```

Comment:
If we do need these Kconfig entries for some reason, maybe we could word them in a way to make it obvious, that these are trivial 1-to-1 copiers, e.g. "support one-to-one copying conversion for 16 bit valid sample size in 32 bit container"

Reply:
sure, good expression